openlibrary/templates/type/list/view.html

```diff
@@ -8,13 +8,9 @@
 $if check_owner and username:
     $ is_owner = username == lst['key'].split('/')[2]
 
-$if check_owner and is_owner:
-    $ template = MyBooksTemplate(username, 'list').render(
-    $     header_title=lst.get('name', 'List'),
-    $     template=render_template("type/list/view_body", lst, is_owner=True),
-    $     page=lst,
+$ template = MyBooksTemplate(lst['key'].split('/')[2], 'list').render(
+    $   header_title= "",
+    $   template=render_template("type/list/view_body", lst, is_owner),
+    $   page=lst
     $ )
-    $:template
-
-$else:
-    $:render_template("type/list/view_body", lst, is_owner)
+$:template
```

Comment:
```suggestion
$:template

```
Add trailing newline to this file.